module/mempool/mock/result_forest.go

```diff
@@ -0,0 +1,109 @@
+// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
this file could be removed too

Reply:
sorry. I already pressed the merge button 🤦

Reply:
I till come in handy when I merge the Sealing Fix I am working on 😉